tools/start-nodes.sh

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env bash
+
+# This script has no arguments
+
+source tools/travis-common-vars.sh
+
+MIM1=${BASE}/_build/mim1/rel/mongooseim
+MIM2=${BASE}/_build/mim2/rel/mongooseim
+MIM3=${BASE}/_build/mim3/rel/mongooseim
+FED1=${BASE}/_build/fed1/rel/mongooseim
```

Comment:
Can we offload `MIM{1,2,3}` & `FED1` variables to `tools/travis-common-vars.sh`?